src/containers/landing/index.tsx

```diff
@@ -1,40 +1,34 @@
 import React, { useEffect } from 'react';
-import { connect, useDispatch } from 'react-redux';
+import { connect, useDispatch, useSelector } from 'react-redux';
 import { Helmet } from 'react-helmet';
 import MapPage from '../../components/mapPageComponents/mapPage';
-import MobileMapPage from '../../components/mapPageComponents/mobileMapPage';
-import MobileLandingBar from '../../components/mapPageComponents/mobileLandingBar';
 import LandingTreeStats from '../../components/landingTreeStats';
 import useWindowDimensions, {
   WindowTypes,
 } from '../../components/windowDimensions';
-import { asyncRequestIsComplete } from '../../utils/asyncRequest';
 import { getMapGeoData } from '../../components/mapPageComponents/ducks/thunks';
 import { LANDING_BODY, LANDING_TITLE } from '../../assets/content';
-import styled from 'styled-components';
-import { MapGeoDataReducerState } from '../../components/mapPageComponents/ducks/types';
 import { C4CState } from '../../store';
-
-interface LandingProps {
-  readonly neighborhoods: MapGeoDataReducerState['neighborhoodGeoData'];
-  readonly blocks: MapGeoDataReducerState['blockGeoData'];
-  readonly sites: MapGeoDataReducerState['siteGeoData'];
-}
-
-const mapStateToProps = (state: C4CState): LandingProps => {
-  return {
-    neighborhoods: state.mapGeoDataState.neighborhoodGeoData,
-    blocks: state.mapGeoDataState.blockGeoData,
-    sites: state.mapGeoDataState.siteGeoData,
-  };
-};
+import { UserAuthenticationReducerState } from '../../auth/ducks/types';
+import { getPrivilegeLevel } from '../../auth/ducks/selectors';
+import { isLoggedIn } from '../../utils/isCheck';
+import styled from 'styled-components';
+import MobileMapPage from '../../components/mapPageComponents/mobileMapPage';
+import MobileLandingBar from '../../components/mapPageComponents/mobileLandingBar';
 
 const PaddedContent = styled.div`
   padding: 24px 50px;
 `;
 
-const Landing: React.FC<LandingProps> = ({ neighborhoods, blocks, sites }) => {
+interface LandingProps {
```

Comment:
🏕️ You dont actually use LandingProps anymore so you can delete this and the connect state to props at the bottom of this component